Behaviors/FrameworkElementBehaviors.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System.Collections.Generic;
+using System.Windows;
+using System.Windows.Data;
+using Thingie.WPF.Converters;
+
+namespace Thingie.WPF.Behaviors
+{
+    /// <summary>
+    /// Represents behaviors that can be applied to a <see cref="FrameworkElement"/>.
+    /// </summary>
+    public class FrameworkElementBehaviors
+    {
+        /// <summary>
+        /// A <see cref="DependencyProperty"/> used to store the element's <see cref="FrameworkElement"/> children that impact the element's <see cref="Visibility"/>. This provides a safe and controlled way to dynamically set the element's properties based on the registered children's properties.
+        /// </summary>
+        public static readonly DependencyProperty VisibilityChildrenDependenciesProperty = 
```

Comment:
Did you mean VisibilityChildrenDependencyProperty?

Reply:
I meant to indicate that there are multiple child dependecies, but if you're okay with VisibilityChildrenProperty, I can change it

Reply:
I'm not sure what a child dependency would be though? Child seems enough to me. Although, I'm not sure why WPF dependency properties are named "dependency properties" either.

Reply:
Okay, I'll rename the property and methods and update the PR

Reply:
Ah, you mean the parent visibility depends on the visibility of its children.

Reply:
Yes, but since you got confused I guess the naming in itself was confusing. Updated the PR.